<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRayon</class>
 <widget class="QDialog" name="DialogRayon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sections Panel</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidgetRayon">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonSupprimer">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonModifier">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageAjout">
    <widget class="QPushButton" name="pushButtonValiderRayon">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>83</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>127</width>
       <height>90</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelLibelleRayon">
        <property name="text">
         <string>Sections label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLibelleRayon"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonImageRayon">
        <property name="text">
         <string>Browse picture</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pageModif">
    <widget class="QPushButton" name="pushButtonModifRayon">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>83</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>127</width>
       <height>90</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="labelLibelleRayonModif">
        <property name="text">
         <string>Sections label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLibelleRayonModif"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonImageRayonModif">
        <property name="text">
         <string>Browse picture</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonAbandonRayon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>83</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelErrorMessageModif">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
